satip.c

```diff
@@ -255,6 +265,33 @@ void cPluginSatip::ParseServer(const char *paramP)
   FREE_POINTER(p);
 }
 
+void cPluginSatip::ParsePortRange(const char *paramP)
+{
+  char *s, *p = skipspace(paramP);
+  char *r = strtok_r(p, "-", &s);
+  unsigned int rangeStart = 0;
+  unsigned int rangeStop = 0;
+  if(r) {
+    rangeStart = strtol(r, NULL, 0);
+    r = strtok_r(NULL, "-", &s);
+    }
+  if(r) {
+    rangeStop = strtol(r, NULL, 0);
+    }
+  else {
+    error("Portrange argument not valid [%s]", paramP);
+    rangeStart = 0;
+    rangeStop = 0;
+    }
+  if(rangeStop - rangeStart < deviceCountM * 2) {
+    error("The given portrange of %d is to small, need >= %d!", rangeStop-rangeStart, deviceCountM*2 );
```

Comment:
In log messages, use the term "port range". Add whitespaces around operators and trim the whitespace after 2.